src/AppBundle/Command/LoadLegislativeDistrictZonesCommand.php

```diff
@@ -0,0 +1,91 @@
+<?php
+
+namespace AppBundle\Command;
+
+use AppBundle\Entity\LegislativeCandidate;
+use AppBundle\Entity\LegislativeDistrictZone;
+use Symfony\Bundle\FrameworkBundle\Command\ContainerAwareCommand;
+use Symfony\Component\Console\Helper\ProgressBar;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+
+class LoadLegislativeDistrictZonesCommand extends ContainerAwareCommand
+{
+    /**
+     * @var \Doctrine\ORM\EntityManager
+     */
+    private $manager;
+
+    protected function configure()
+    {
+        $this
+            ->setName('app:load:legislative-district-zones')
```

Comment:
Shall we name it `app:legislatives:load-district-zones` instead? It sounds more logical to me.